bpf/process/types/basic.h

```diff
@@ -854,6 +854,21 @@ FUNC_INLINE void copy_reverse(__u8 *dest, uint len, __u8 *src, uint offset)
 	}
 }
 
+#ifdef __LARGE_BPF_PROG
+FUNC_INLINE void file_copy_reverse(__u8 *dest, uint len, __u8 *src, uint offset)
+{
+	uint i;
+
+	len &= STRING_POSTFIX_MAX_MASK;
+#pragma unroll
```

Comment:
Since you have a `__LARGE_BPF_PROG` macro around this, you should be able to just drop the pragma unroll and use bounded loops here.